modules/stanford_courses_importer/config/install/config_pages.type.stanford_courses_importer.yml

```
@@ -0,0 +1,27 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - config_pages_overrides
+third_party_settings:
+  config_pages_overrides:
+    455b050b-d304-4c74-acf4-f04767664a6b:
+      field: su_explore_course_url
+      delta: 0
```

This will use every value and build an array instead of only using the first delta.
```suggestion
      delta: -1
```

---

fixed